packages/muelu/src/Interface/MueLu_HierarchyManager.hpp

```diff
@@ -93,6 +93,7 @@ namespace MueLu {
         doPRrebalance_          (MasterList::getDefault<bool>("repartition: rebalance P and R")),
         implicitTranspose_      (MasterList::getDefault<bool>("transpose: use implicit")),
         fuseProlongationAndUpdate_ (MasterList::getDefault<bool>("fuse prolongation and update")),
+        sizeOfMultiVectors_     (MasterList::getDefault<int>("number of vectors")),
```

Comment:
`numberOfMultiVectors_`?

Reply:
It's the number of individual vectors in a MultiVector.  I wasn't really happy with any of the names I came up with.  The current name is at least consistent with the usage/name in MueLu::Hierarchy.

Reply:
Yep, not sure either. Maybe `numberOfRHS`?